### ISSUE TITLE
Invalid Message Catalogs Generated

### ISSUE THREAD

**Issue by fny** [lingui/js-lingui#753]
Using 3.0.0-13 to 3.0.0-16 after an extract and compile, I'm getting the following error:

```
Argument of type '{ messages: { "API.greenlight_status.invalid_email": string; "API.sessions.invalid_email": string; "API.sessions.invalid_password": string; "API.sessions.invalid_phone": string; "DashboardPage.child_care_footer": string; ... 156 more ...; "{0, plural, one {child} other {children}}": (string | { ...; })[][]; }; }' is not assignable to parameter of type 'Record<string, CompiledMessage>'.
  Property 'messages' is incompatible with index signature.
    Type '{ "API.greenlight_status.invalid_email": string; "API.sessions.invalid_email": string; "API.sessions.invalid_password": string; "API.sessions.invalid_phone": string; "DashboardPage.child_care_footer": string; ... 156 more ...; "{0, plural, one {child} other {children}}": (string | { ...; })[][]; }' is not assignable to type 'CompiledMessage'.
      Object literal may only specify known properties, and '"API.greenlight_status.invalid_email"' does not exist in type '(string | [string, (string | undefined)?, (string | Object | undefined)?])[]'.  TS2345
     3 | import en from './locales/en/messages.js'
       |
     5 | 
     6 | export const i18n = setupI18n()
  >  7 | i18n.load('en', en)
       |                 ^
     8 | i18n.load('es', es)
     9 | i18n.activate('en')
```

**Comment by tricoder42** [comment]
Please create a repository with minimal example to reproduce this bug. Otherwise it's hard to help without knowing your full setup.

**Comment by fny** [comment]
Will do.

**Comment by fny** [comment]
Here is the repository: https://github.com/fny/bugreports-lingui

**Comment by fny** [comment]
Yeah this looks like a bug that occurs when importing using the simple instead of po.

**Comment by semoal** [comment]
The problem is you're importing messages.js as default, you must extract the messages
```jsx
import { messages as enMessages } from './locales/en/messages.js'
      
export const i18n = setupI18n()
i18n.load('en', enMessages)
```

**Comment by fny** [comment]
This doesn't work either.

**Comment by semoal** [comment]
The fix for typescript inference is not deployed yet. https://github.com/lingui/js-lingui/commit/93b14953b99203f47a14be5afb75f19b4bd0b6a6#comments

Once deployed, I'll quote you, no worries.

**Comment by tricoder42** [comment]
It was just published in `3.0.0-20`
